src/domain/eventSearch/SearchResultList.tsx

```diff
@@ -102,6 +102,26 @@ const SearchResultList: React.FC<Props> = ({
     window.scrollTo(0, 0);
   };
 
+  const clearFilters = () => {
+    const search = getSearchQuery({
+      categories: [],
+      dateTypes: [],
+      districts: [],
+      endDate: null,
+      isCustomDate: !!(startDate || endDate),
+      keywords: [],
+      places: [],
+      publisher: null,
+      search: "",
+      startDate: null,
+      targets: []
+    });
+
+    push({ pathname: `/${locale}/events`, search });
+    // Scroll to top when changing filters. Ignore this on SSR becasue window doesn't exist
+    window.scrollTo(0, 0);
```

Comment:
The comment here describes that the window line should be ignored when using SSR, but there doesn't seem to be any explicit condition that does that. As far as I understand, with this code, the `scrollTo` action would not be ran on client side though.

Reply:
So true, I'll remove scrollTo